tests/src/test/java/alluxio/client/cli/fsadmin/command/DoctorCommandIntegrationTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fsadmin.command;
+
+import alluxio.cli.fsadmin.command.DoctorCommand;
+import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests for doctor command.
+ */
+public final class DoctorCommandIntegrationTest extends AbstractFsAdminShellTest {
+  @Test
+  public void masterNotRunning() throws Exception {
+    mLocalAlluxioCluster.stopMasters();
+    mFsAdminShell.run("doctor");
+    String expected = "The Alluxio leader master is not currently serving requests.";
+    Assert.assertThat(mErrOutput.toString(), CoreMatchers.containsString(expected));
+  }
+
+  @Test
+  public void doctorCategoryInvalid() {
+    mFsAdminShell.run("doctor", "invalidCategory");
+    String expected = String.format("%s%n%s%n%s%n",
```

Comment:
Joiner.on("%n").join(...) + "%n" is easier to extend and easier visually

Reply:
Changed to 
```
    String expected = String.join("\n",
        DoctorCommand.usage(),
        DoctorCommand.description(),
        "doctor category is invalid.") + "\n";
    Assert.assertEquals(expected, mOutput.toString());
```
Thanks!